lisa_commands/lambda_function.py

```diff
@@ -39,3 +45,29 @@ def lambda_handler(data, _context):
 
     request.submit_slack_request(data, chat_action)
     return request.return_status()
+
+def valid_slack_request(data):
+    """
+    Did this request come from slack
+    data - the data object (dict)
+    """
+    slack_signature = data['headers']['X-Slack-Signature']
+    slack_request_timestamp = data['headers']['X-Slack-Request-Timestamp']
+    request_body = data['body']
+    basestring = f"v0:{slack_request_timestamp}:{request_body}".encode('utf-8')
+
+    #Make the Signing Secret a bytestring too.
```

Comment:
Super nitpick 9000: Technically per PEP8, there should be a space between # comments' start and bodies. (Also a space in front of them, but that's moot for comment-only lines like this).